mozilla-mobile/fenix/app/src/main/res/values/strings.xml

```diff
@@ -1454,12 +1454,14 @@
     <string name="top_sites_max_limit_confirmation_button">OK, Got It</string>
 
     <!-- DEPRECATED STRINGS -->
```

Comment:
@eliserichards What would be best here would be to just delete all those strings. Does that make sense to you as well?
thanks

Reply:
@BranescuMihai: I think Elise is out, would you mind removing all the deprecated strings so I can expose all the strings this weekend to localizers please? (so, the ones landed by Elise as well as the new one from https://github.com/mozilla-l10n/android-l10n/pull/249/commits/7fa41a2516299a5dddc75d55e1025fd9f45d2e0f)